### PR TITLE
bpo-43178: Redundant method overrides in queue

### PR DESCRIPTION
**Pull request by pelegshilo**
Fixed redundant method definitions of _qsize.

https://bugs.python.org/issue43178

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept this contribution by verifying everyone involved has signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

## Recognized GitHub username

We couldn't find a [bugs.python.org](https://bugs.python.org/) (b.p.o) account corresponding to the following GitHub usernames:

@pelegshilo

This might be simply due to a missing "GitHub Name" entry in one's b.p.o account settings. This is necessary for legal reasons before we can look at this contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.




You can [check yourself](https://check-python-cla.herokuapp.com/) to see if the CLA has been received.

Thanks again for the contribution, we look forward to reviewing it!

File: Misc/NEWS.d/next/Library/2021-02-09-10-06-37.bpo-43178.Lbvvsz.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Removed redundant method definitions of _qsize in PriorityQueue and LIFOQueue. These methods already inherit the exact same implementation from their parent class, Queue, so these are not necessary
```
Comment (pablogsal) [inline Misc/NEWS.d/next/Library/2021-02-09-10-06-37.bpo-43178.Lbvvsz.rst]: Without considering the code itself, there is no need for a NEWS entry as this change is not user-facing.

Comment (pelegshilo) [reply Misc/NEWS.d/next/Library/2021-02-09-10-06-37.bpo-43178.Lbvvsz.rst]: Sorry. This is my first pull request, I have been trying to follow the tutorial closely. I will pay more attention in the future

Comment (pelegshilo) [reply Misc/NEWS.d/next/Library/2021-02-09-10-06-37.bpo-43178.Lbvvsz.rst]: Do I need to fix this?

Comment (pablogsal) [reply Misc/NEWS.d/next/Library/2021-02-09-10-06-37.bpo-43178.Lbvvsz.rst]: > Sorry. This is my first pull request, I have been trying to follow the tutorial closely. I will pay more attention in the future

Is fine, don't worry :)

Normally you need to add a NEWS file, but this is one of the cases were this is not needed.

>Do I need to fix this?

Yeah, please, delete this file

**Review by pablogsal — APPROVED** [review]

**Pull request closed without merging**